Detectors/ZDC/simulation/src/Detector.cxx

```diff
@@ -1479,7 +1479,7 @@ void Detector::createCsideBeamLine()
   // Inner trousers
   TGeoCompositeShape* pIntTrousersC = new TGeoCompositeShape("intTrousersC", "QCLint:ZDCC_c1+QCLint:ZDCC_c2");
   // Volume: QCLint
-  TGeoMedium* medZDCvoid = gGeoManager->GetMedium("ZDC_VoidNoField");
+  TGeoMedium* medZDCvoid = gGeoManager->GetMedium("ZDC_VoidNoField$");
```

Comment:
can you remind me what the purpose of the `$` character is?

Reply:
You should better ask the authors of the code; but as the framework does not strip these characters from the names, of the $ is omitted in the name, the medium is not found. Geant4 gives an exception when you try to build a volume without a material.

Reply:
Ahh I see ... it was just a spelling error (I thought it had a deeper meaning).
In fact, this shows that one should use the MatManager class to retrieve the media. This would have thrown an error.